layouts/shortcodes/config-uri-azure-oidc.html

```diff
@@ -0,0 +1 @@
+https://raw.githubusercontent.com/kubeflow/manifests/v1.1-branch/kfdef/kfctl_azure_aad.v1.1.0.yaml
```

Comment:
Can we move this shortcode to an Azure specific folder and add OWNERS there?
In that case, Azure approvers can approve on their own.

Reply:
Sounds good @Bobgy Thanks for the suggestions

@eedorenko @sudivate - will you be able to move the shortcode? Then I/we can add OWNERS (a copy of what's in `/docs/azure/`)